wiki/BanchoBot/th.md

```diff
@@ -0,0 +1,179 @@
+---
+tags:
+  - bancho
+  - server bot
+  - commands
+  - announcements
+---
+
+# BanchoBot
+
+*ระวังสับสนกับ [Bancho (เซิฟเวอร์แชท)](/wiki/Bancho_(server))*
+
+![รูปหน้าผู้ใช้ของ BanchoBot](img/BanchoBot.jpg "รูปหน้าผู้ใช้ของ BanchoBot")
+
+**BanchoBot** (หรืออีกชื่อหนึ่งว่า *Bancho*) เป็นแชทบอทสำหรับช่วยเหลือผู้เล่นผ่านทางช่องแชทในเกม osu! โดยจะบอกข้อมูลที่เกี่ยวข้องกับเกม อาทิ จำนวนเกมที่เล่น จำนวนครั้งที่เริ่มใหม่ ฯลฯ และจะรับคำสั่งบางคำสั่ง
+
+BanchoBot เขียนขึ้นโดย [Echo](https://osu.ppy.sh/users/431) และ BanchoBot ก็เป็นผู้ให้บริการ [Bancho IRC](/wiki/Internet_Relay_Chat) (ไออาร์ซี)
+
+BanchoBot มีทั้ง [หน้าผู้ใช้ของ osu!](https://osu.ppy.sh/users/3) และ [บัญชีผู้ใช้ทวิตเตอร์](https://twitter.com/banchoboat)
+
+## คำสั่ง
+
+*สำหรับคำสั่งที่ใช้ภายในเกม ดูที่ [หน้าต่างพูดคุย](/wiki/Chat_Console#commands-list)*
+
+BanchoBot สามารถตอบสนองต่อคำสั่งผ่านข้อความเฉพาะในช่องแชท ทุกคำสั่งของจะขึ้นต้นด้วยเครื่องหมายอัศเจรีย์ (`!`) ตามด้วยชื่อคำสั่งโดยไม่มีเว้นวรรค (ตัวอักษรตัวพิมพ์ใหญ่หรือตัวพิมพ์เล็กไม่มีผลต่อคำสั่ง) โดยคำสั่งเหล่านั้นสามารถใช้ได้ในช่องแชทสาธารณะและช่องแชทส่วนตัวกับ BanchoBot
+
+หากผู้ใช้ทั่วไปสั่งคำสั่งผ่านทางช่องแชทสาธารณะ ผู้ใช้อื่นจะไม่เห็นคำสั่งและการตอบกลับแต่อย่างใด ซึ่งการตอบกลับจะส่งผ่านทางช่องแชทส่วนตัวของ BanchoBot แทน
+
+ผู้ใช้สามารถใช้คำสั่ง `/bb` เพื่อเรียกช่องแชทส่วนตัวของ BanchoBot ได้
```

Comment:
```suggestion
หากผู้ใช้ทั่วไปสั่งคำสั่งผ่านทางช่องแชทสาธารณะ ผู้ใช้อื่นจะไม่เห็นคำสั่งและการตอบกลับแต่อย่างใด ซึ่งการตอบกลับจะส่งผ่านทางช่องแชทส่วนตัวของ BanchoBot แทนและ ผู้ใช้สามารถใช้คำสั่ง `/bb` เพื่อเรียกช่องแชทส่วนตัวของ BanchoBot ได้
```
The en articles mention about using /bb to open the chat with Bancho AND send the command immediately (/bb <command>)
Doing /bb alone would do nothing so you should add that part